pwnlib/libcdb.py

```diff
@@ -612,6 +619,8 @@ def search_by_build_id(hex_encoded_id, unstrip=True):
             Hex-encoded Build ID (e.g. 'ABCDEF...') of the library
         unstrip(bool):
             Try to fetch debug info for the libc and apply it to the downloaded file.
+        offline(bool):
+            Enable offline search mode, which will disable online providers.
```

Comment:
This sounds like an xor, like you either have offline or online mode, but not both. Can you please rephrase this to clearify this is an exclusive offline mode but offline providers are used too when the offline parameter is False.

That makes me think the parameter might be better named `offline_only` or similar.